### DISCUSSION TITLE
[@types/markdown-it] Should not depend on `highlight.js`

### DISCUSSION THREAD

**Discussion by Lemmingh** [DefinitelyTyped/DefinitelyTyped#56187] [Issues with a @types package] [answered]
The dependency on `highlight.js` was introduced in #49293, where @WilsonGramer stated that `@types/markdown-it` depends on types from `highlight.js`.

Now we still have:

<https://github.com/DefinitelyTyped/DefinitelyTyped/blob/f346df47bf74d9c5572343c97d58c4b2a632323b/types/markdown-it/package.json#L3-L5>

However,

* **Nothing** in `@types/markdown-it` references `highlight.js`.

* `markdown-it` can work perfectly without `highlight.js`, and has **never** depended on it. (There is only a demo that uses `highlight.js`)

---

It's likely to be a **mistake**. Besides, we can see from the history that the existence of `highlight.js` or `@types/highlight.js` caused many unexpected problems.

**`highlight.js` should be removed from `@types/markdown-it`.**

[a comment by an automated account is not shown]

**Reply by peterblazejewicz** [reply]
It appears in JSDoc, so people added it into a test. I'll rewrite tests to not use that dependency, as extraneous
TbD
